cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc

```diff
@@ -93,3 +94,7 @@ a[coalesce($lower,0)..coalesce($upper,size(a))]
 * Arithmetic expressions containing `null`: `1 + null`
 * Function calls where any arguments are `null`: `sin(null)`
 
+[[is-null-is-not-null]]
+== Using `IS NULL` and `IS NOT NULL`
+Testing any value against `null` with both the `=` and the `<>` operators always evaluates to `null`.
```

Comment:
This sounds to me like you were talking about applying both operators at once.
```suggestion
Testing any value against `null` with either the `=` or the `<>` operator always evaluates to `null`.
```

Reply:
```suggestion
Testing any value against `null`, either with the `=` operator or with the `<>` operator, always evaluates to `null`.
```